bindgen/lib.rs

```diff
@@ -704,6 +704,10 @@ fn rust_to_clang_target(rust_target: &str) -> Box<str> {
         let idx = clang_target.rfind('-').expect(TRIPLE_HYPHENS_MESSAGE);
 
         clang_target.replace_range((idx + 1).., "elf");
+    } else if clang_target.ends_with("apple-ios-sim") {
```

Comment:
Nit: Please make this general enough to include `*-apple-tvos-sim`, `*-apple-watchos-sim` etc.

Reply:
Done. I cleaned up the code a bit by splitting the "triple" and reassembling it at the end.